### PR TITLE
Add license of hashgrid, cc #9248

### PR DESCRIPTION
**Pull request by yangwanting**
https://github.com/dotjay/hashgrid/blob/master/LICENSE

### CODE REVIEW

File: ajax/libs/hashgrid/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "version": "6",
   "description": "A little tool that inserts a layout grid in web pages, allows you to hold it in place, and toggle between displaying it in the foreground or background.",
   "homepage": "http://hashgrid.com/",
+  "license":　"Apache-2.0",
```
Comment (x09326) [inline ajax/libs/hashgrid/package.json]: @yangwanting 
There is a unvalid space.
Please modify it.
And remember write the correct library name in commit message next time.

**Comment by x09326** [comment]
LGTM @cdnjs/intern2

**Comment by x09326** [comment]
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**